.github/workflows/packer-build-ami.yml

```diff
@@ -0,0 +1,41 @@
+name: Packer build AWS AMI's 
+on: 
+  workflow_dispatch:
+    branches:
+      - prod
+
+jobs:
+  plan:
+    environment: Terraform
+    defaults:
+      run:
+        working-directory: /home/runner/work/ballerine/deploy/aws_ami
+    runs-on: ubuntu-latest
+    name: Packer build Artifacts
+    steps:
+      - name: Checkout to Git
+        uses: actions/checkout@v2
+  
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
```

Comment:
Can we replace auth by having assume role ?
https://github.com/aws-actions/configure-aws-credentials

an article about it: https://faun.pub/configure-aws-credentials-for-github-actions-with-assume-role-14a21926c1f6